pkg/reconciler/openshift/tektonaddon/extension.go

```diff
@@ -120,21 +117,25 @@ func (oe openshiftExtension) PostReconcile(ctx context.Context, comp v1alpha1.Te
 			}
 
 			if err := createInstallerSet(ctx, oe.operatorClientSet, addon, manifest, oe.version,
-				miscellaneousResourcesInstallerSet, "addon-openshift"); err != nil {
+				MiscellaneousResourcesInstallerSet, "addon-openshift"); err != nil {
```

Comment:
retrun `RCONCILE_AGAIN_ERR` here to ensure that this API call has created the installerSet.